### PR TITLE
first stub of stats about plugins

### PR DESCRIPTION
**Pull request by catscanner**

### CODE REVIEW

File: tests/test_server.py [hunk 1]
```diff
@@ -55,3 +55,8 @@ def test_server_stats_returns_api_statistics(flask_app):
     assert response_json["deployments"]["last_7_days"] == 0
     assert response_json["deployments"]["today"] == 0
     assert response_json["deployments"]["yesterday"] == 0
+    assert response_json["plugins"]["all"] == {}
+    assert response_json["plugins"]["last_60_mins"] == {}
+    assert response_json["plugins"]["last_7_days"] == {}
+    assert response_json["plugins"]["today"] == {}
+    assert response_json["plugins"]["yesterday"] == {}
```
Comment (balintzs) [inline tests/test_server.py]: What do the empty dicts mean?

Comment (catscanner) [inline tests/test_server.py]: Similar to the 0s for the deployments, they're the empty sets of plugins.
if populated they'd contain  {"plugin name": number of times called, ...} etc

Comment (balintzs) [inline tests/test_server.py]: Thanks. Could you, then, add test(s) where those dicts aren't empty?

**Comment by balintzs** [comment]
LGTM
